draft/oss-support/post.md

```diff
@@ -0,0 +1,70 @@
+# OSS Support
+
+Free software. It's not _really_ free. Time, effort, innovation and labor go
+into developing the open source projects that we use and sometimes can't live
+without. A lot of these projects live in the background, known only to those
```

Comment:
> A lot of these projects live in the background, known only to those that dig into the low level building blocks of the net or in the lower depths of the application layer of open and closed operating systems

This isn't exactly a _run-on_, but it is a bit wordy. Maybe revisit this section with fresh eyes.